tracer.go

```diff
@@ -136,6 +136,9 @@ func NewTracer(
 	if hostname, err := os.Hostname(); err == nil {
 		t.tags = append(t.tags, Tag{key: TracerHostnameTagKey, value: hostname})
 	}
+	if t.hostIPv4 != 0 {
```

Comment:
See comment on L126 - we already on the new data model, makes sense to fix this.